src/skmultiflow/data/file_stream.py

```diff
@@ -39,7 +39,8 @@ class FileStream(Stream):
     >>> # Imports
     >>> from skmultiflow.data.file_stream import FileStream
     >>> # Setup the stream
-    >>> stream = FileStream('skmultiflow/data/datasets/sea_stream.csv')
+    >>> stream = FileStream("https://raw.githubusercontent.com/scikit-multiflow/"
+    ...                     "master/sea_stream.csv")
```

Comment:
I believe the link is incomplete (the repository containing the datasets is missing).

Reply:
Fixed in e8b8f3fd3bb60bd1e6ad657e3fe45da11ac0a8d2